### ISSUE TITLE
Issue on page /core/matplotlib.html

### ISSUE THREAD

**Issue by kitmacleod** [ProjectPythia/pythia-foundations#112]
If for complete Python beginners, why Matplotlib? Huge respect to John Hunter and the community, but it is a low level API compared to [Seaborn](https://seaborn.pydata.org/#), [Plotly](https://plotly.com/) and others. I understand why Matplotlib is a key part of the scientific Python stack (after coming to it ~ 13 years ago from Matlab). I also love how Hadley and others have used grammar of graphics etc for the impressive ggplot API.

**Comment by dopplershift** [comment]
> Matplotlib is a key part of the scientific Python stack

That's exactly why Matplotlib. 😉 It's also used as the basis for plotting by several other tools like MetPy, Geocat, Cartopy, etc.

That doesn't mean it has to be the *only* plotting library covered. We'd be happy to have contributions of training materials for other plotting libraries with earth system science applications.

**Comment by clyne** [comment]
Indeed. We're iterating over some examples for the [GeoCAT gallery](https://github.com/NCAR/GeoCAT-examples) using Holoviews+Datashader to demonstrate interactive plots and the ability to handle high resolution data. I'd like to get these incorporated somehow in Pythia. Thanks for the input, @kitmacleod !

**Comment by clyne** [comment]
> This might be another example of a Cookbook topic!

Coincidentally, NCAR's GeoCAT team just created their first interactive plotting example using Holoviews+datashader+geofviews [here](https://geocat-examples.readthedocs.io/en/latest/gallery-notebooks/Datashader/MPAS_Datashader_Trimesh.html).  Is this what you have in mind for a cookbook, @brian-rose
